src/components/WebexSettings/WebexSettings.stories.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import WebexSettings from './WebexSettings';
+
+export default {
+  title: 'Meetings/Webex Settings',
+  component: WebexSettings,
+};
+
+const Template = (args) => <WebexSettings {...args} />;
+
+export const Settings = Template.bind({});
```

Comment:
@lalli-flores Known issues:
- momentum tabs don't look good on small mobiles
- we will format the tabs to look like the ones in the image in the user story in a separate task, as it is an application wide feature: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-236060

Reply:
I think we should move away from Momentum, knowing that we will soon not support it